plugins/rpmdb-consistency.sh

```diff
@@ -1,7 +1,11 @@
 #!/bin/bash
 
 run_checks() {
-	zypper --no-refresh --quiet verify --dry-run
+	# check rpm DB itself
+	rpm -D "%_rpmlock_path /run/rpmdb"  --verifydb
```

Comment:
```suggestion
	rpm -D "%_rpmlock_path /run/rpmdb" --verifydb
```

and IMO this needs a comment why it sets `%_rpmlock_path`